ibis/backends/dask/helpers.py

```diff
@@ -0,0 +1,235 @@
+from __future__ import annotations
+
+import itertools
+from typing import Callable
+
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+
+# def asseries(value, size=1):
+#     """Ensure that value is a pandas Series object, broadcast if necessary."""
+#     if isinstance(value, dd.Series):
+#         return value
+#     elif isinstance(value, pd.Series):
+#         return dd.from_pandas(value)
+#     elif isinstance(value, (list, np.ndarray)):
+#         return dd.from_pandas(itertools.repeat(np.array(value), size))
+#     else:
+#         return dd.from_array(itertools.repeat(value, size))
+#         #return dd.from_pandas(pd.Series(np.repeat(value, size)))
+from dask.base import DaskMethodsMixin
+
+from ibis.util import gen_name
+
+
+def asseries(value, like=None):
+    """Ensure that value is a pandas Series object, broadcast if necessary."""
+
+    if isinstance(value, dd.Series):
+        return value
+    elif isinstance(value, dd.core.Scalar):
+        # Create a Dask array from the scalar
+        array = da.full((1,), value, chunks=(1,))
```

Comment:
@jcrist @ncclementi do you know whether this is triggering a `.compute()`? The point of this function would be to *broadcast* the input value as a `dd.Series`, preferably without triggerint compute.

Reply:
It shouldn't. Are you seeing anything suspicious that makes you think otherwise? 
I did a quick check just in case: 
```python
In [2]: import dask.array as da
In [3]: import dask.dataframe as dd

In [4]: value = 5 #value is a scalar right?

In [5]: array = da.full((1,), value, chunks=(1,))

In [6]: array
Out[6]: dask.array<full_like, shape=(1,), dtype=int64, chunksize=(1,), chunktype=numpy.ndarray>
```

Reply:
Well, value is a dask scalar, not a plain python literal.

Reply:
It should still be alright, here is an example with a dask scalar.
```python
In [13]: import dask

In [14]: ddf = dask.datasets.timeseries()

In [15]: ddf.x.mean()
Out[15]: dd.Scalar<series-..., dtype=float64>

In [16]: val = ddf.x.mean()

In [17]: array = da.full((1,), val, chunks=(1,))

In [18]: array
Out[18]: dask.array<full_like, shape=(1,), dtype=float64, chunksize=(1,), chunktype=numpy.ndarray>

In [19]: isinstance(val, dd.core.Scalar)
Out[19]: True
```

Reply:
That does compute (it's going through the `__array__` protocol, which is likely a bug/misfeature).

I think what you want here is:


```suggestion
        array = da.from_delayed(value.to_delayed(), (1,), dtype=value.dtype)
```

This is a bit of a roundabout way to cast a scalar to a single-partition series, but given the existing API I'm not sure there's a better way.

Reply:
Oh interesting, I didn't see that coming. I was assuming the what's on Output line 18, implied this was lazy, bu I guess not. My bad.

Reply:
Thanks! Already incorporated your suggestion.